### DISCUSSION TITLE
no (expensive) motion sensors, but only camera & frigate to control the light ?

### DISCUSSION THREAD

**Discussion by schwarmik** [blakeblackshear/frigate#3407] [General]
Hello all,
I have a "beginner's question" - I want to install a camera on each side of my house. In addition, actually two KNX motion detectors to switch the outdoor light. But I could actually do without these expensive detectors, if I could detect people and cars via the cameras and Frigate, to then just switch the outside light. The logic to switch the light is done by my home automation software (edomi), I just need the "person/car detected" trigger data point via mqtt.

Can I implement it this way ? Or are the latency times possibly too high?

[Raspberry 4 with docker and frigate] -- [mqtt] -- [edomi Software with Logic to switch the lights]

thx

**Comment by NickM-27** [comment]
I have frigate controlling lights outside, 6 months in it still works great and I've had literally 0 mistriggers

**Reply by DrSpaldo** [reply]
> I have frigate controlling lights outside, 6 months in it still works great and I've had literally 0 mistriggers

@NickM-27 I would be interested to know what (if) you have changed from the default settings for person detection? I am doing the same, however, still manage to get false positives every day or so...

@schwarmik , it is defiantly possible, I am using it for my lights automation via home assistant. Sometimes it is super snappy, othertimes it may take a few seconds to fire. I am not really sure why as it would take some serious testing

**Reply by NickM-27** [reply]
Are they false positives that happen on the same false object every time?

I just have zones for driveway and front yard (which are required for events and to turn the lights on) and then just a couple object masks at the base of the trees in the yard. With that I have had no person false positives at all.

**Reply by schwarmik** [reply]
Thanks for the answers - this sounds promising and I will try it out. The next step is the order of the camera, probably it will be a Hikvison, if I understood it correctly, this must be able to provide only the stream via RTSP. Or do you have a recommendation?

**Reply by NickM-27** [reply]
There's recommendations in the doc that cover what to look for https://docs.frigate.video/hardware
